keras_hub/src/models/whisper/whisper_audio_converter.py

```diff
@@ -1,14 +1,10 @@
-import numpy as np
+import keras.ops as ops
+import tensorflow as tf
```

Comment:
We can never do a bare import of tf like this. Check other files in the library.